lib/private/Collaboration/Collaborators/MailPlugin.php

```diff
@@ -175,7 +175,7 @@ public function search($search, $limit, $offset, ISearchResult $searchResult) {
 
 						if ($this->shareeEnumeration) {
 							try {
-								$cloud = $this->cloudIdManager->resolveCloudId($contact['CLOUD'][0]);
+								$cloud = $this->cloudIdManager->resolveCloudId($contact['CLOUD'][0]??'');
```

Comment:
```suggestion
								$cloud = $this->cloudIdManager->resolveCloudId($contact['CLOUD'][0] ?? '');
```

I'm wondering if we shouldn't instead do a continue when the element doesn't exist

Reply:
I think we should, but a good sanity check is needed anyway